src/Hosting/Abstractions/src/MetricsPrototype/IMeterFactory.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Extensions.Metrics;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning disable RS0016 // Add public types and members to the declared API
+public sealed class MeterFactoryOptions
+{
+    public string? ContainerId { get; set; }
+    public IDictionary<string, string> DefaultTags { get; } = new Dictionary<string, string>();
```

Comment:
>IDictionary [](http://example.com/codeflow?start=11&length=11)

why this is IDictionary and not IEnumerable?

Reply:
The app or library configures the options. They need a way to add default tags which are then passed to the meter.